test/unit/org/apache/cassandra/config/ParseAndConvertUnitsTest.java

```diff
@@ -63,11 +63,11 @@ public void testConfigurationLoaderParser()
         assertEquals(DurationSpec.inSeconds(86400), config.trace_type_query_ttl);
         assertEquals(DurationSpec.inSeconds(604800), config.trace_type_repair_ttl);
         assertEquals(DurationSpec.inMilliseconds(2000), config.permissions_validity);
-        assertEquals(DurationSpec.inMilliseconds(0), config.permissions_update_interval);
+        assertEquals(null, config.permissions_update_interval);
         assertEquals(DurationSpec.inMilliseconds(2000), config.roles_validity);
-        assertEquals(DurationSpec.inMilliseconds(0), config.roles_update_interval);
+        assertEquals(null, config.roles_update_interval);
         assertEquals(DurationSpec.inMilliseconds(2000), config.credentials_validity);
-        assertEquals(DurationSpec.inMilliseconds(0), config.credentials_update_interval);
+        assertEquals(null, config.credentials_update_interval);
```

Comment:
nit: Happens in a few places in the patch, but could just use...

```suggestion
        assertNull(config.credentials_update_interval);
```